components/workflow-mgt/org.wso2.carbon.identity.workflow.mgt/src/main/java/org/wso2/carbon/identity/workflow/mgt/bean/WorkflowRequest.java

```diff
@@ -100,4 +100,12 @@ public void setCreatedBy(String createdBy) {
 
         this.createdBy = createdBy;
     }
+
+    public void setOperationType(String string) {
+        this.eventType = string;
```

Comment:
```suggestion
        this.operationType = string;
```

change variable name